elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -108,6 +108,11 @@ public function action__plugins_loaded() {
 		if ( apply_filters( 'debug_bar_enable', false ) || apply_filters( 'wpcom_vip_qm_enable', false ) ) {
 			// Load ElasticPress Debug Bar
 			require_once __DIR__ . '/../../debug-bar-elasticpress/debug-bar-elasticpress.php';
+
+			// And ensure the logging has been setup (since it also hooks on plugins_loaded)
```

Comment:
That's more or less what I did, except for the `function_exists` check, which I had not thought of. 🤭

However, I just tested it and could not see any queries being logged.

Reply:
As discussed on Slack, working as intended.